### ISSUE TITLE
Warning "Old Style Heading" on list continuation followed by open block

### ISSUE THREAD

**Issue by OliverO2** [asciidoctor/asciidoctor-intellij-plugin#228]
AsciiDoc Plugin Version: 0.21.4

IDE Name and Version: IntelliJ IDEA 2018.2.6 (Ultimate Edition)
Build #IU-182.5107.16, built on November 6, 2018
JRE: 1.8.0_152-release-1248-b22 amd64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Linux 4.15.0-39-generic

Using a list continuation symbol `+` followed by an open block start `--` incorrectly generates a "WARNING AsciiDoc Old Style Heading".

Example taken from the [Asciidoctor User Manual 20.2.1. List continuation](https://asciidoctor.org/docs/user-manual/#list-continuation):

```
* The header in AsciiDoc must start with a document title.
+
--
Here's an example of a document title:

----
= Document Title
----

NOTE: The header is optional.
--
```

Possibly related: #185

**Comment by ahus1** [comment]
Thanks for reporting this with an accurate example. This will be fixed in the next release.

**Comment by ahus1** [comment]
This available in 0.22.0. As this is still a pre-release (and it contains other experimental features) is available only from the [GitHub releases page](https://github.com/asciidoctor/asciidoctor-intellij-plugin/releases) of this project.

**Comment by OliverO2** [comment]
Thanks for the quick fix. Just checked: Works for me!
